src/test/java/spoon/test/variable/VariableTest.java

```diff
@@ -37,6 +42,58 @@
 
 public class VariableTest {
 
+    @Test
+    public void testSetType() {
+        // contract: one can use setType without having a very hard generics type checking error
+
+        // to get this done, we use the refactoring below
+        CtClass c = Launcher.parseClass("class C{ void f() {Object i=null; }}");
+        List<CtLocalVariable> localVariables = c.getElements(new TypeFilter<>(CtLocalVariable.class));
+        System.out.println(localVariables);
```

Comment:
I'm not a massive fan of system outs in test cases. This only fills the log. How about removing this?

Reply:
done